app/build.gradle.kts

```diff
@@ -52,6 +52,6 @@ tasks.test {
 java {
     withSourcesJar()
     toolchain {
-        languageVersion.set(JavaLanguageVersion.of(17))
+        languageVersion.set(JavaLanguageVersion.of(19))
```

Comment:
scheduled executor only impl's AutoClosable starting in 19...

Reply:
TIL